### ISSUE TITLE
Unable to run test suite using PhantomJS on Mac OS X 

### ISSUE THREAD

**Issue by roelrz** [linkedin/venus.js#309]
We are unable to run a test suite (e.g., 100 files) using PhantomJS on Mac OS X successfully.

The test suite runs for a while but at a certain point we get the following the console:

```
error:  PhantomJS unable to load test URL: http://127.0.0.1:2013/venus-core/41 (timeout)
error:  PhantomJS unable to load test URL: http://127.0.0.1:2013/venus-core/42 (timeout)
error:  PhantomJS unable to load test URL: http://127.0.0.1:2013/venus-core/43 (timeout)
```

**Comment by tgvrssanthosh** [comment]
We ran into this issue on linux boxes as well.

**Comment by tgvrssanthosh** [comment]
raised a pull request https://github.com/linkedin/venus.js/pull/362

**Comment by tgvrssanthosh** [comment]
merged #363.
